.expeditor/update_hab_release.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Updates the Habitat version everywhere in this cookbook so it works with new
+# versions of Habitat.
+#
+set -evx
+
+if [[ -z $VERSION ]]; then
+  echo "ERROR: Version undefined, cannot continue, exiting"
+  exit 1
+fi
+
+branch="expeditor/${VERSION}"
+git checkout -b ${branch}
+
+sed -i "s/%r{^hab .*\/}/%r{^hab ${VERSION}\/}/" test/integration/config/default_spec.rb
+sed -i "s/%r{^hab .*\/}/%r{^hab ${VERSION}\/}/" test/integration/package/default_spec.rb
+sed -i "s/HAB_VERSION = '.*'.freeze/HAB_VERSION = '${VERSION}'.freeze/" resources/install.rb
+sed -i "s/^- Habitat version: .*/- Habitat version: ${VERSION}/" README.md
+sed -i "s/^version '.*'/version '${VERSION}'/" metadata.rb
```

Comment:
Can we add a VERSION file that also get's updated with the value? That's what Expeditor is looking for to power things like `update_changelog`

Reply:
line 22!